src/routes/auth.route.ts

```diff
@@ -1,21 +1,71 @@
 import express, { Request, Response } from "express";
+import superagent from "superagent";
 
 import { register } from "../services/auth.service";
 
 const authRouter = express.Router();
 
-// authRouter.route("/twitter").get(
-// post request to Twitter API
-// receive redirectUri in response
-// redirect your user to received redirectUri res.redirect()
-// );
+const redirectUri = `http://localhost:${process.env.PORT}/api/auth/twitter-callback`;
```

Comment:
```suggestion
const redirectUri = `http://localhost:${process.env.PORT}/api/auth/twitter-callback`;
```
move all redirectUri to process.env, after deploy it can be at other path not localhost

Reply:
is done